config/database.yml

```diff
@@ -0,0 +1,85 @@
+# PostgreSQL. Versions 9.1 and up are supported.
```

Comment:
You should to ignore this file. Create `database.example.yml` file instead.

Reply:
Commented lines were removed, and added new database.example file.